api/internal/entity/card.go

```diff
@@ -0,0 +1,13 @@
+package entity
+
+type Card struct {
+	ID          string `json:"id"`
+	Surname     string `json:"surname"`
+	Name        string `json:"name"`
+	Patronymic  string `json:"patronymic"`
+	PhoneNumber string `json:"phone_number"`
+	City        string `json:"city"`
+	Street      string `json:"street"`
+	Zip         string `json:"zip_code"`
+	Discount    int    `json:"discount"`
+}
```

Comment:
Rename `Card` -> `Customer`. Customer is more clear than Card. Also rename all files `card.go` -> `customer.go`